src/templates/storyViewer.handlebars

```diff
@@ -46,7 +46,7 @@
     <div lang="{{localizedMessages.language}}" class="sjrk-st-story-viewer-footer-container">
         <div>
             <a href="javascript:void(0)" class="sjrkc-st-story-viewer-previous sjrk-st-story-viewer-previous">
-                <img src="/src/img/icons/icon-back.svg" alt="left arrow icon">
+                <img src="/src/img/icons/icon-back.svg" alt="{{localizedMessages.message_goBackIcon}}" aria-hidden="true">
```

Comment:
The value of "aria-hidden" should be flipped programmatically.

Reply:
The same comment applies to the similar changes in `src/templates/storyEditor.handlebars`.

Reply:
My intention was to hide the additional info in the accessibility tree related to this link, as any alt-text for this image would be identical to the text displayed or would be otherwise redundant, superfluous or even potentially distracting or confusing. I.e. if the alt-text value for the image is "left arrow icon", NVDA says "left arrow icon graphic back link" rather tha

Reply:
Reviewing the WAI-ARIA recommendations, it cautions against using `aria-hidden` in this way without first considering the accessibility ramifications of the decision, but I believe this is a change for the better. I can certainly change it back if you feel it isn't appropriate.

Reply:
I agree this is a change for the better. But as the "back" button is only hidden at the first publishing step, it makes more sense to programmatically control the true/false value of "aria-hidden" to be in sync with the showing and hiding of the "back" button.